File: src/main/java/com/spring/sun/test/Person.java
package com.spring.sun.test;

public class Person {
	
	private int id;
	private String name;
	
	public Person() {
		
	}
	
	public Person(int id, String name) {
		this.id = id;
		this.name = name;
	}
	
	@Override
	public String toString() {
		return "Person [id=" + id + ", name=" + name + "]";
	}

	public void speak() {
		System.out.println("Hello, I am a person!");
	}
}

File: Cpp/CppBeginnerTutorial/CppBeginnerTutorial/Person.cpp
//
//  Person.cpp
//  CppBeginnerTutorial
//
//  Created by Yu Sun on 5/19/16.
//  Copyright © 2016 Yu Sun. All rights reserved.
//

#include "Person.hpp"

Person::Person() {
    name = "Sun";
}

string Person::toString() {
    return "Person's name is: " + name;
}

void Person::setName(string newName) {
    name = newName;
}

string Person::getName() {
    return name;
}